src/bucket/Bucket.cpp

```diff
@@ -6,31 +6,31 @@
 // ASIO is somewhat particular about when it gets included -- it wants to be the
 // first to include <windows.h> -- so we try to include it before everything
 // else.
-#include "util/asio.h"
```

Comment:
I think this will cause an issue for Windows compilation/development.
We have set SortIncludes to false in .clang-format to make sure this doesn't happen.

It seemed like it was also possible to set the weight of a specific include so it's always on top.